src/Take.Blip.Builder/FlowManager.cs

```diff
@@ -321,7 +324,7 @@ private static bool ValidateDocument(LazyInput lazyInput, InputValidation inputV
                     return DateTime.TryParse(lazyInput.SerializedContent, out _);
 
                 case InputValidationRule.Regex:
-                    return Regex.IsMatch(lazyInput.SerializedContent, inputValidation.Regex);
+                    return Regex.IsMatch(lazyInput.SerializedContent, inputValidation.Regex, RegexOptions.Compiled | RegexOptions.IgnoreCase, _regexTimeout);
```

Comment:
The input validation is dynamic because the user input it. Compiling this regex will load our CPU

Reply:
Done. Using RegexOption.None as default